src/mds/MDSRank.cc

```diff
@@ -2541,10 +2549,11 @@ MDSRankDispatcher::MDSRankDispatcher(
 
 bool MDSRankDispatcher::handle_command(
   const cmdmap_t &cmdmap,
-  bufferlist const &inbl,
+  MCommand *m,
   int *r,
   std::stringstream *ds,
-  std::stringstream *ss)
+  std::stringstream *ss,
+  bool *need_reply)
```

Comment:
I generally like to set these out params to a known value at function start; otherwise we rely on the caller initializing them correctly. (In which case, we need to document that contract in the header.)